includes/class-sensei-course-results.php

```diff
@@ -109,7 +104,7 @@ public function content() {
 		global $wp_query, $woothemes_sensei, $current_user;
 
 		if( isset( $wp_query->query_vars['course_results'] ) ) {
-			$woothemes_sensei->frontend->sensei_get_template( 'course-results/course-info.php' );
+            Sensei_Templates::get_template( 'course-results/course-info.php' );
```

Comment:
@dwainm - Why not make a `sensei_get_template()` function instead? May be easier for folks to implement.